hyperspy/drawing/image.py

```diff
@@ -261,6 +262,7 @@ def plot(self, **kwargs):
 
         if self.colorbar is True:
             self._colorbar = plt.colorbar(self.ax.images[0], ax=self.ax)
+            self._colorbar.set_label(self.quantity_label)
```

Comment:
Probably just a matter of taste: `set_label(self.quantity_label, rotation=-90)`?